### DISCUSSION TITLE
Import generic JSON

### DISCUSSION THREAD

**Discussion by JohnTMurphy-NIU** [ArcadeData/arcadedb#398] [Q&A]
A note in the ArcadeDB documentation says that there is support for importing generic JSON files: [https://docs.arcadedb.com/#Importer](https://docs.arcadedb.com/#Importer). I can't find a worked example. I'm trying to import a collection of documents from [Semantic Scholar](https://api.semanticscholar.org/corpus/download/); the 'sample' files are only a few records and good to test with. The files are JSON-L, but could be easily converted to standard JSON. What would be the procedure to import these into a DB? I'm an ArcadeDB newbie, so my starting point is a clean install. Thanks!

**Comment by lvca** [comment]
A little late, but we finally have the JSON importer: https://docs.arcadedb.com/#JSON-Importer!

**Reply by amcsSH** [reply]
Dear @lvca , thank you - awesome work. Are there more examples of using the JSON-Importer in addition to the shared link?
I'm looking forward to hearing back from you.
Best, Andreas

**Reply by lvca** [reply]
We ned to improve our docs. For now. check out the test https://github.com/ArcadeData/arcadedb/blob/3dabad0478bebd52c5f75a430fa918f3e336903a/integration/src/test/java/com/arcadedb/integration/importer/JSONImporterIT.java#L35.
